crates/sui-source-validation/src/lib.rs

```diff
@@ -8,19 +8,20 @@ use move_compiler::compiled_unit::CompiledUnitEnum;
 use move_core_types::account_address::AccountAddress;
 use move_package::compilation::compiled_package::CompiledPackage;
 use move_symbol_pool::Symbol;
+use sui_sdk::error::RpcError;
 
 use sui_sdk::rpc_types::{SuiRawData, SuiRawMoveObject, SuiRawMovePackage};
 use sui_types::{base_types::ObjectID, error::SuiError};
 
 #[cfg(test)]
 mod tests;
 
-type ReadApi = sui_sdk::ReadApi;
+type ReadApi = sui_sdk::apis::ReadApi;
```

Comment:
This type originally existed so that in `sim_test` mode we could use the other API.  That seems to no longer be necessary?  If so, it can be changed into a `use`.